app/src/main/res/values-ar/strings.xml

```diff
@@ -523,4 +523,5 @@
   <string name="profile_edit_fragment_test_activity_label">نشاط اختبار جزء تحرير ملف التعريف</string>
   <string name="administrator_controls_fragment_test_activity_label">يتحكم المسؤول في نشاط اختبار التجزئة</string>
   <string name="revision_navigation_cards_header">أكمل دراستك</string>
+  <string name="profile_edit_activity_title">"تحرير ملف التعريف"</string>
```

Comment:
Remove this string. It will get added by other team. We(developers) only add strings for english language.